src/controller/webhook/webhook.controller.ts

```diff
@@ -84,4 +93,19 @@ export class WebhookController implements interfaces.Controller {
             appLogger.error("Error in pushing to webhook", { url, data, action });
         }
     }
+  }
+
+  private async webhookCall(data: any, action: string): Promise<any> {
```

Comment:
`webhookCall` - why this function is added, it's already there. Duplicate function